src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/MicrosoftAccountStartup.cs

```diff
@@ -78,6 +78,7 @@ public override void ConfigureServices(IServiceCollection services)
 }
 
 [RequireFeatures("OrchardCore.Deployment")]
+[Feature(MicrosoftAuthenticationConstants.Features.AAD)]
```

Comment:
I feel something goes wrong here, why do we need AAD in Microsoft Account?

Reply:
Because this need to be enabled when the `Deployment` feature is enabled + it should be part of the `MicrosoftAuthenticationConstants.Features.AAD` feature since it will not work without the features that would be enabled by `MicrosoftAuthenticationConstants.Features.AAD` feature.

Reply:
@hishamco any follow up questions or objections. I like to merge this one as part of 2